tests/e2e/__snapshots__/cli.test.ts.snap

```diff
@@ -68,7 +68,7 @@ exports['CLI should output multiple bundles result as html 1'] = `
2 2.8%","originalName":"bar.js","data":{"$area":104}},{"name":"foo.js \u2022 97 B \u2022 2.6%","originalName":"foo.js","data":{"$area":97}}]},{"name":"[sourceMappingURL] \u2022 35 B \u2022 0.9%","originalName":"[sourceMappingURL]","data":{"$area":35}},{"name":"[unmapped] \u2022 1 B \u2022 0.0%","originalName":"[unmapped]","data":{"$area":1}},{"name":"[EOLs] \u2022 1 B \u2022 0.0%","originalName":"[EOLs]","data":{"$area":1}}]},{"name":"data/inline-map.js \u2022 2.93 KB \u2022 80.7%","originalName":"data/inline-map.js","data":{"$area":2999},"children":[{"name":"../../node_modules/browser-pack/_prelude.js \u2022 480 B \u2022 12.9%","originalName":"../../node_modules/browser-pack/_prelude.js","data":{"$area":480}},{"name":"src \u2022 201 B \u2022 5.4%","originalName":"src","data":{"$area":201},"children":[{"name":"bar.js \u2022 104 B \u2022 2.8%","originalName":"bar.js","data":{"$area":104}},{"name":"foo.js \u2022 97 B \u2022 2.6%","originalName":"foo.js","data":{"$area":97}}]},{"name":"[sourceMappingURL] \u2022 2.26 KB \u2022 62.3%","originalName":"[sourceMappingURL]","data":{"$area":2316}},{"name":"[unmapped] \u2022 1 B \u2022 0.0%","originalName":"[unmapped]","data":{"$area":1}},{"name":"[EOLs] \u2022 1 B \u2022 0.0%","originalName":"[EOLs]","data":{"$area":1}}]}]}},"1":{"name":"data/foo.min.js","data":{"name":"/ \u2022 718 B \u2022 100.0%","originalName":"/","data":{"$area":718},"children":[{"name":"../../node_modules/browser-pack/_prelude.js \u2022 480 B \u2022 66.9%","originalName":"../../node_modules/browser-pack/_prelude.js","data":{"$area":480}},{"name":"src \u2022 201 B \u2022 28.0%","originalName":"src","data":{"$area":201},"children":[{"name":"bar.js \u2022 104 B \u2022 14.5%","originalName":"bar.js","data":{"$area":104}},{"name":"foo.js \u2022 97 B \u2022 13.5%","originalName":"foo.js","data":{"$area":97}}]},{"name":"[sourceMappingURL] \u2022 35 B \u2022 4.9%","originalName":"[sourceMappingURL]","data":{"$area":35}},{"name":"[unmapped] \u2022 1 B \u2022 0.1%","originalName":"[unmapped]","data":{"$area":1}},{"name":"[EOLs] \u2022 1 B \u2022 0.1%","originalName":"[EOLs]","data":{"$area":1}}]}},"2":{"name":"data/inline-map.js","data":{"name":"/ \u2022 2.93 KB \u2022 100.0%","originalName":"/","data":{"$area":2999},"children":[{"name":"../../node_modules/browser-pack/_prelude.js \u2022 480 B \u2022 16.0%","originalName":"../../node_modules/browser-pack/_prelude.js","data":{"$area":480}},{"name":"src \u2022 201 B \u2022 6.7%","originalName":"src","data":{"$area":201},"children":[{"name":"bar.js \u2022 104 B \u2022 3.5%","originalName":"bar.js","data":{"$area":104}},{"name":"foo.js \u2022 97 B \u2022 3.2%","originalName":"foo.js","data":{"$area":97}}]},{"name":"[sourceMappingURL] \u2022 2.26 KB \u2022 77.2%","originalName":"[sourceMappingURL]","data":{"$area":2316}},{"name":"[unmapped] \u2022 1 B \u2022 0.0%","originalName":"[unmapped]","data":{"$area":1}},{"name":"[EOLs] \u2022 1 B \u2022 0.0%","originalName":"[EOLs]","data":{"$area":1}}]}}};
```

Comment:
Highlight abnormal changes here:

```
- "children":[{"name":"foo.min.js \u2022 718 B \u2022 19.3%"
+ "children":[{"name":"data/foo.min.js \u2022 718 B \u2022 19.3%"
```

I'm not sure why. Could anyone help me please?

Reply:
I guess it's because old implementation uses `getCommonPathPrefix` and `data` prefix is deleted

Reply:
Updated. It should be fine now.